cmd/collector/app/handler/grpc_handler.go

```diff
@@ -19,6 +19,7 @@ import (
 
 	"go.uber.org/zap"
 	"google.golang.org/grpc/codes"
+	_ "google.golang.org/grpc/encoding/gzip" // enable gzip compression support
```

Comment:
I think the linter error is from the [old `golint` implementation](https://github.com/golang/lint) which has been deprecated in favour of `go vet` and `staticcheck`. I tested PR this with `golangci-lint` and does not gripe about this line having a blank import.

If we want to avoid including this comment, we could try waiting for #3237 to be merged (if it's approved) and rebase?

Reply:
Thank you!

I'll wait for the #3237, and then update this PR to the original version.